spec/factories/assignments.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :assignment do
+    user_id 1
```

Comment:
This factory is a little funky because it doesn't use the `team` or `user` associations, nor does it set a `team_id`.